client/src/utils/BGraphQueryUtil.ts

```diff
@@ -148,3 +149,132 @@ export function bioNodeOutDegree (bgraphPathQuery, clause: Filter): any {
     return false;
   });
 }
+
+// EPI PATH QUERIES
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export function epiPathPre (bgraphPathQuery, clause: Filter): any {
```

Comment:
Query functions should no longer be exported as they are meant to be private to the BGraphQueryUtil